content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+id: 20
+title: 'Email Tabs'
+slug: email-tabs
+is_featured: false
+platforms: ['addon']
+min_release: 62
+thumbnail: /static/images/experiments/email-tabs/email-tabs.png
+description: >
+  Easily create emails from all of your tabs and make saving and sharing easier than ever.
+introduction: >
+  <p>Ever needed to save or share a whole bunch of tabs as you research, shop, or just browse the web? Email tabs lets you create beautiful emails from your open tabs to save them for later or share them. You can use Email Tabs to automatically send along links, screenshots, or even the text from articles.<p>
+  <p>Email tabs currently works with the Gmail webmail client, but we’re working to bring it to other popular webmail providers as well.</p>
+image_twitter: /static/images/experiments/notes/email-tabs-twitter.jpg
+image_facebook: /static/images/experiments/notes/email-tabs-fb.jpg
```

Comment:
My linter (see #3919) says that your Facebook and Twitter icons are 404ing. Which seems plausible since I don't even see them in this commit.

```
Email Tabs
  checking images:
    - Missing /static/images/experiments/notes/email-tabs-fb.jpg
    - Missing /static/images/experiments/notes/email-tabs-twitter.jpg
  checking urls:
    - https://discourse.mozilla-community.org/c/test-pilot/email-tabs { status: 'dead', statusCode: 404 }
```

And the Discourse 404 seems legit since it presumably wouldn't be created until release (but will be good to check once this lands on stage).

Reply:
also path is wrong `/experiments/notes/`

Reply:
@meandavejustice good catch